.github/workflows/fronts.yml

```diff
@@ -0,0 +1,34 @@
+name: DCR Fronts Ophan tag status
+on:
+  # Triggers the workflow on push and pull request events but only for the main branch
```

Comment:
What do you mean push _**and pull**_? Is the comment explaining anything that isn't intrinsic to actions?